src/test/java/hudson/plugins/copyartifact/DownstreamBuildSelectorTest.java

```diff
@@ -31,7 +31,7 @@
 import java.util.Map;
 import java.util.Set;
 
-import hudson.plugins.copyartifact.testutils.CopyArtifactUtil;
+import hudson.plugins.copyartifact.testutils.CopyArtifactUtil;

```

Comment:
You can see changed lines in "Files changed" tabs and will find lines marked changed even they looks unchanged. Those lines were ended with LF.
Anyway, don't mind those changes as I described.